packages/core/src/components/List/index.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import childrenWithComponentName from '../../prop-types/childrenWithComponentName';
+import withStyles, { WithStylesProps } from '../../composers/withStyles';
+import Item from './Item';
+
+export { Item };
+
+export type Props = {
+  /** List items. */
+  children: NonNullable<React.ReactNode>;
+  /** Flex align the items. */
+  flexAlign?: boolean;
```

Comment:
Maybe call this `inline`?

Reply:
i went with `flexAlign` because `inline` usually applies `inline-block`. how's that sound?

Reply:
i went with `horizontal` instead - thoughts?

Reply:
👍